src/data_provider/src/sharedDefs.h

```diff
@@ -106,4 +108,24 @@ enum MacOSArchitecture
     ARM64
 };
 
+static const std::set<std::string> UNIX_PYPI_DEFAULT_BASE_DIRS
+{
+    "/usr/lib/python*/*-packages",
+    "/usr/lib64/python*/*-packages",
+    "/usr/local/lib/python*/*-packages",
+    "/home/*/.local/lib/python*/*-packages",
+    "/opt/homebrew/lib",
+    "/Libary/Python",
+};
+
+static const std::set<std::string> UNIX_NPM_DEFAULT_BASE_DIRS
+{
+    "/usr/local/lib",
+    "/opt/homebrew/lib",
+    "/usr/lib",
+    "/home/*/.npm-global/lib",
+    "/Users/*/.npm-global/lib",
+    "/home/*/.nvm/versions/node/v*/lib",
```

Comment:
Why isn't this location included as default paths? 
![image](https://github.com/wazuh/wazuh/assets/13010397/8afac7df-aca7-4105-bce2-095507cc21c7)

Reply:
related to C++ standard definition.